client/packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -89,9 +91,10 @@ const DataRowComponent = <T extends RecordWithId>({
                 ? theme => alpha(theme.palette.secondary.main, 0.1)
                 : null,
               '&.MuiTableRow-root': {
-                '&:nth-of-type(even)': {
-                  backgroundColor: 'background.toolbar',
-                },
+                backgroundColor:
```

Comment:
Forcing the colour to be one or the other so it's consistent throughout the app =/